projects/premake5.lua

```diff
@@ -55,7 +56,147 @@ CreateWorkspace({name = "voxelate"})
 	project("fastlz")
 		language("C")
 		kind("StaticLib")
+		warnings "Off"
 		files({
 			"../fastlz/fastlz.c",
 			"../fastlz/fastlz.h",
 		})
+
+    project("reactphysics3d")
+        language "C++11"
+        kind "StaticLib"
+		symbols "On"
+		warnings "Off"
+
+        includedirs {
+            "../reactphysics3d/src/",
+        }
+
+        files {
+            "../reactphysics3d/src/configuration.h",
```

Comment:
I ported it's cmake file to premake. It's premake file didn't use globs, but you're right that premake supports globs. But is there any reason to deviate from that and use globs anyway? What's the point, apart from a reduced premake file size?